### ISSUE TITLE
No SSO option

### ISSUE THREAD

**Issue by demonfoxkiller** [PlayFab/UnityEditorExtensions#126]
the Unity Editor Extension has no way of signing in with Microsoft account via SSO.

**Comment by MichaelTGood** [comment]
Yeah, I have the same issue, and obviously, this breaks my game. I had been testing with an account I made with my personal email, but am switching to the company email/PF account now, and I can't login in Unity for the game. 

Is there a solution or work around for this, so I can continue building? Or do I have to make another PF account that isn't SSO (not the preferred solution, obviously)?

I also have the same issue with [PlayFab/vscode-playfab-explorer](https://github.com/PlayFab/vscode-playfab-explorer).
